applications/openshift/api-server/audit_log_forwarding_enabled_logging_api/rule.yml

```diff
@@ -0,0 +1,52 @@
+
+title: Ensure that Audit Log Forwarding Is Enabled
+
+description: |-
+  OpenShift audit works at the API server level, logging all requests coming to the server.
+  Audit is on by default and the best practice is to ship audit logs off the cluster for retention.
+  The cluster-logging-operator is able to do this with the <pre>ClusterLogForwarders</pre> resource.
+  The forementioned resource can be configured to logs to different third party systems.
```

Comment:
nit: s/logs/log/ ?

Reply:
@rhmdnd I kept the "legacy" `logging.openshifit.io` rules pointing to 4.16.
From 4.17 there is no mention of this API anymore.